src/components/TopBar/TopBar.tsx

```diff
@@ -36,28 +36,29 @@ interface ITopBarProps {
 export const TopBar = ({ children, id, type, icon, hideCloseButton = false }: React.PropsWithChildren<ITopBarProps>) => {
   const [isOpen, setIsOpen] = useState<boolean>(true);
 
-  const statusLocalStorageId = useMemo(() => `${id}-status`, [id]);
-  const textLocalStorageId = useMemo(() => `${id}-text`, [id]);
+  const statusLocalStorageKey = useMemo(() => `${id}-status`, [id]);
+  const textLocalStorageKey = useMemo(() => `${id}-text`, [id]);
 
+  // TODO: change to local storage?
```

Comment:
Personally, I think keeping session storage is a good choice for the top bar, specifically so that users will see the warning again when they open a new tab or visit the UI again.  If we change to local storage, then users will not be able to see the top bar any more; they may miss some important message just because they hide the top bar once. I would suggest we have another hook like `useSessionStorage` for this case.

Reply:
@matedo1 what do you think?

Reply:
I generalized the hook into `useStorage`